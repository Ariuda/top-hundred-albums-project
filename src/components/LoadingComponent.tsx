import React from 'react';

const LoadingComponent: React.FC = () => {
    return (
        <div className="loading">
            <h1>Top 100 albums in the iTunes Store</h1>
            <div className="animation">
                <div className="bar bar-one"></div>
                <div className="bar bar-two"></div>
                <div className="bar bar-three"></div>
                <div className="bar bar-four"></div>
                <div className="bar bar-five"></div>
                <div className="bar bar-six"></div>
                <div className="bar bar-seven"></div>
                <div className="bar bar-eight"></div>
                <div className="bar bar-nine"></div>
                <div className="bar bar-ten"></div>
            </div>
            Loading...
        </div>
    )
};

export default LoadingComponent;