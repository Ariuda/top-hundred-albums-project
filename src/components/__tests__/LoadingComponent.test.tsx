import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import { LoadingComponent } from '../LoadingComponent';

it('should render the loading component', () => {
    const wrapper = shallow(<LoadingComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});