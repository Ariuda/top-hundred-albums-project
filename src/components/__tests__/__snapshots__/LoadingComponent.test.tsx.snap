// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the loading component 1`] = `
<div
  className="loading"
>
  <h1>
    Top 100 albums in the iTunes Store
  </h1>
  <div
    className="animation"
  >
    <div
      className="bar bar-one"
    />
    <div
      className="bar bar-two"
    />
    <div
      className="bar bar-three"
    />
    <div
      className="bar bar-four"
    />
    <div
      className="bar bar-five"
    />
    <div
      className="bar bar-six"
    />
    <div
      className="bar bar-seven"
    />
    <div
      className="bar bar-eight"
    />
    <div
      className="bar bar-nine"
    />
    <div
      className="bar bar-ten"
    />
  </div>
  Loading...
</div>
`;
